src/vc_util_js/src/lib.rs

```diff
@@ -20,10 +22,19 @@ pub fn validate_verified_adult_presentation(
         issuer_canister_id: Principal::from_slice(issuer_canister_id),
         issuer_origin: String::from_utf8(issuer_origin.to_vec()).expect("wrong issuer origin"),
     };
-    vc_util::custom::validate_verified_adult_presentation(
+    let credential_spec = {
```

Comment:
So this function will be removed as well at some point? And allow to pass through the credential spec from JS? Or is `vc_util_js` never intended to become the JS equivalent of `vc_util`?

Reply:
Yes, the function will be replaced with the generic validator that takes a spec.  But the js module is not indented to be fully equivalent to `vc_util`, at least for now.